scripts/clean_up.py

```diff
@@ -0,0 +1,112 @@
+import os
+import glob
+import shutil
+import tarfile
+import argparse
+
+from qcore import utils
+
+SUBMISSION_DIR_NAME = 'submission_temp'
+SUBMISSION_SUBDIR_NAME = 'slurm_and_logs'
+SUBMISSION_TAR = 'submission.tar'
+SUBMISSION_FILES = ['flist_LF', 'slurm_header.cfg', 'machine_env.sh', 'submit.sh', '*.template']
+SUBMISSION_SL_LOGS = ['*.sl', '*.err', '*.out']
+
+LF_DIR_NAME = 'LF_temp'
+LF_SUB_DIR_NAME = 'OutBin'
+LF_TAR = 'LF.tar'
+LF_FILES = ['Rlog', 'Restart', 'SlipOut']
+
+
+def make_tar(source_dir, out_dir):
+    """params: source_dir:source dir of all files to tar
+               out_dir: output dir for the tar.gz
+    """
+    try:
+        print("start making tar")
+        with tarfile.open(out_dir,'w:gz') as tar:
+            tar.add(source_dir,arcname=os.path.basename(source_dir))
+            print("finished making csv tar")
+    except Exception as e:
+        print(e)
+        print("cant'make tar")
+
+
+def move_files(sim_dir, dest_dir, file_list):
+    """
+    move files from sim dir to dest dir
+    :param sim_dir: path to source realization folder, eg. /home/melody.zhu/Albury/Runs/Albury/Albury_HYP15-21_S1384
+    :param dest_dir: path to destination dir
+    :param file_list: a list of files/file_pattern to copy
+    :return:
+    """
+    for f in file_list:
+        for p in glob.glob1(sim_dir, f):
+            try:
+                shutil.move(os.path.join(sim_dir, p), os.path.join(dest_dir, p))
+            except Exception as e:
+                print("erro while copy ing file from {} to {}\n{}".format(sim_dir, dest_dir, e))
+
+
+def create_temp_dirs(sim_dir, outer_dir_name, inner_dir_name):
+    """
+    creates four temp dirs containing files to be tared
```

Comment:
from my understanding this creates two nested directories and returns their paths?